src/main/java/racingcar/dao/RacingInsertDao.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.dao;
+
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import racingcar.domain.Car;
+
+import javax.sql.DataSource;
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+@Repository
+public class RacingInsertDao {
+    private static SimpleJdbcInsert simpleJdbcInsert;
+
+    public RacingInsertDao(DataSource dataSource) {
+        this.simpleJdbcInsert = new SimpleJdbcInsert(dataSource)
+                .withTableName("PLAY_RESULT")
+                .usingGeneratedKeyColumns("id");
+    }
+
+    public static void insert(int trialCount, List<String> winnerList, Car car) {
```

Comment:
```suggestion
    public static void insert(PlayResult playResult) {
```
`PLAY_RESULT`가 가진 컬럼들을 관리하는 `PlayResult` 도메인이 있으면 좀 응집성이 생길것 같아요! 🤔 
테스트 용이성도 좋아질것 같구요!